languages/ruby/exercises/concept/instance-variables/.meta/design.md

```diff
@@ -0,0 +1,38 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the concept instance variables and "nil" in Ruby.
+
+## Learning objectives
+
+- Know that objects can store state using instance variables
+- Know that instance variables should be treated as private and to use getter and setter methods
+- Know that `nil` is a Ruby object used to represent "nothingness"
+- Know that instance variables are `nil` before they are assigned otherwise
+
+## Out of scope
+
+- Boolean values (True/False)
+- Truthy/Falsey-ness
+
+## Concepts
+
+- `instance-variables`
+- `nil`
+
+## Prerequisites
+
+- `basics`
+
+## Resources
+
+### After
+
+- [The nil value in Ruby](https://medium.com/rubycademy/the-nil-value-in-ruby-d60e6a3642b9#:~:text=method%20implementation-,The%20nil%20value,%E2%80%9Clack%20of%20an%20object%E2%80%9D.&text=Unlike%20other%20languages%2C%20the%20nil,the%20non%2Dinstantiable%20NilClass%20class.)
+
+## Representer
+
+This exercise does not require any specific representation logic to be added to the [representer](https://github.com/exercism/ruby-representer).
+
+## Analyzer
+
+This exercise does not require any specific analyzer logic to be added to the [analyzer](https://github.com/exercism/ruby-analyzer).
```

Comment:
You could consider removing these sections as they don't have any useful contents.